Chapter06/recipe-04/c-example/version.h.in

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define VERSION_MAJOR @VERSION_MAJOR@
```

Comment:
What do you think about showing how to use `cmakedefine` in conjuction with `configure_file` in this example?

Reply:
Cool! Did not know about that. Good idea.